src/main/scala/org/constellation/p2p/Download.scala

```diff
@@ -426,12 +412,23 @@ class DownloadProcess[F[_]: Concurrent: Timer: Clock](
       .flatMap(_ => snapshotsProcessor.processSnapshots(hashes))
       .flatTap(_ => dao.metrics.updateMetricAsync("downloadSecondPassComplete", "true"))
 
-  private def finishDownload(snapshot: SnapshotInfo): F[Unit] =
+  private def removeSnapshotInfoFromDisk(snapshotHeight: Long,
+                                         overWritePath: String = dao.snapshotInfoPath.pathAsString) = {
+    val snapshotInfoAtHeight = dao.snapshotInfoPath.collectChildren(_.isDirectory)
+      .filter { file =>
+        val Array(height, hash) = file.name.split('_')
```

Comment:
If `file.name` is absolute then it fails if someone uses `_` in path.
I have already written such function in `PeerAPI` which is the safer version. We can reuse it by extracting it to another service instead of copy-pasting. It is more or less:

`file.name.split('/').reverse.head` -> it returns you the last segment of path so it ignores what is before the directory name

Reply:
Awesome